src/libs/rover/scheduler.cpp

```diff
@@ -651,31 +658,25 @@ void Scheduler<FloatType>::to_blueprint(conduit::Node &dataset)
       conduit::Node &n_int = dataset["fields/intensities"];
       n_int["topology"] = topo_name;
       n_int["association"] = "element";
+      n_int["units"] = "intensity units";
       vtkm::cont::ArrayHandle<FloatType> ints = m_result.flatten_intensities();
       FloatType *ints_buffer = get_vtkm_ptr(ints);
       // can't set external since this goes out of scope
       n_int["values"].set(ints_buffer, ints.GetNumberOfValues());
-      n_int["shape"].set(shape);
       n_int["strides"].set(strides);
-      n_int["labels"].append() = "groups";
-      n_int["labels"].append() = "width";
-      n_int["labels"].append() = "height";
     }
 
     if(m_result.has_optical_depth(0))
     {
       conduit::Node &n_op = dataset["fields/optical_depth"];
       n_op["topology"] = topo_name;
       n_op["association"] = "element";
+      n_op["units"] = "path length metadata";
```

Comment:
Same thing as the "intensity units". We will also have to decide if this is a good default value.